elements/pl-number-input/pl-number-input.py

```diff
@@ -183,21 +183,51 @@ def parse(element_html, data):
         data['format_errors'][name] = 'No submitted answer.'
         data['submitted_answers'][name] = None
         return
-
-    # Convert to float or complex
-    try:
-        a_sub_parsed = pl.string_to_number(a_sub, allow_complex=allow_complex)
-        if a_sub_parsed is None:
-            raise ValueError('invalid submitted answer (wrong type)')
-        if not np.isfinite(a_sub_parsed):
-            raise ValueError('invalid submitted answer (not finite)')
+        # Accept fractional input, then convert it to a float or decimal
+        # (only works for certain integer addition/subtraction problems)
+    elif '/' in a_sub:
+        a_sub = a_sub.strip()
+        a_sub_list = a_sub.split('/')
```

Comment:
What if there is more than one `'/'` character in the string?

Reply:
There will be an error message that shows after the user grades their answer 
<img width="555" alt="screen shot 2019-02-23 at 5 44 02 pm" src="https://user-images.githubusercontent.com/43425272/53293070-b7ce1500-3792-11e9-8c00-e0edabb39ec7.png">

Reply:
Entering `20/5/10` is accepted as a valid number, which it should not be.